wpeverest/user-registration
# UR-1601 Enhancement  - Don't allow required fields to delete

### All Submissions:






### Changes proposed in this Pull Request:



Closes # Previously when `Disable Default WordPress Login Screen` is enabled but `Redirect Default WordPress Login To` is not selected settings were saved. This PR solves this issue.

### How to test the changes in this Pull Request:

1.Email field should not be able to delete.
2.Password field should be only be able to delete when auto-generate-password is enabled.


### Types of changes:




### Other information:




### Changelog entry

>  Enhancement  - Don't allow required fields to delete
